### PR TITLE
Fix fips-bundle-test artifactId (it is for github repo lib-fips-bundle-test, looks confusing but in fact it's not :)

### PR DESCRIPTION
**Pull request by olamy**
Signed-off-by: Olivier Lamy <olamy@apache.org>

# Link to GitHub repository

# When modifying release permission

List the GitHub usernames of the users who should have commit permissions below:
- `@username1`
- `@username2`

This is needed in order to cut releases of the plugin or component.

If you are modifying the release permission of your plugin or component, fill out the following checklist:

```[tasklist]
### Release permission checklist (for submitters)
- [ ] The usernames of the users added to the "developers" section in the .yml file are the same the users use to log in to [accounts.jenkins.io](https://accounts.jenkins.io/).
- [ ] All users added have logged in to [Artifactory](https://repo.jenkins-ci.org/) and [Jira](https://issues.jenkins.io/) once.
- [ ] I have mentioned an [existing team member](https://github.com/orgs/jenkinsci/teams) of the plugin or component team to approve this request.
```

## When enabling automated releases (cd: true)

Follow the [documentation](https://www.jenkins.io/doc/developer/publishing/releasing-cd/) to ensure, your pull request is set up properly. Don't merge it yet.  
In case of changes requested by the hosting team, an open PR facilitates future reviews, without derailing work across multiple PRs.

### Link to the PR enabling CD in your plugin

```[tasklist]
### CD checklist (for submitters)
- [ ] I have provided a link to the pull request in my plugin, which enables CD according to the documentation. 
```

```[tasklist]
### Reviewer checklist
- [ ] Check that the `$pluginId Developers` team has `Admin` permissions while granting the access.
- [ ] In the case of plugin adoption, ensure that the Jenkins Jira default assignee is either removed or changed to the new maintainer.
- [ ] If security contacts are changed (this includes add/remove), ping the security officer (currently `@Wadeck`) in this pull request. If an email contact is changed, wait for approval from the security officer.
```
There are [IRC Bot commands](https://jenkins.io/projects/infrastructure/ircbot/#issue-tracker-management) for it.

### CODE REVIEW

**Comment by olamy** [comment]
https://github.com/jenkinsci/lib-fips-bundle-test/pull/3

**Review by timja — APPROVED** [review]

File: permissions/component-fips-bundle-test.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name: "lib-fips-bundle-test"
 github: "jenkinsci/lib-fips-bundle-test"
 paths:
-  - "io/jenkins/test/fips/lib-fips-bundle-test"
```
Comment (olamy) [inline permissions/component-fips-bundle-test.yml:5]: weird 
```
Caused by: org.apache.maven.wagon.authorization.AuthorizationException: authorization failed for https://repo.jenkins-ci.org/releases/io/jenkins/test/fips/fips-bundle-test/20.vb_e5df22f62ea_/fips-bundle-test-20.vb_e5df22f62ea_.pom, status: 403 Forbidden
```

here https://github.com/jenkinsci/lib-fips-bundle-test/actions/runs/10845721879/job/30097173717


@timja

Comment (olamy) [reply permissions/component-fips-bundle-test.yml:5]: oh maybe I need to learn patience? :)

Comment (timja) [reply permissions/component-fips-bundle-test.yml:5]: should run quick but the job does take a bit of time: https://github.com/jenkins-infra/repository-permissions-updater/blob/master/Jenkinsfile#L16

Comment (olamy) [reply permissions/component-fips-bundle-test.yml:5]: no worries, Friday arvo I will have a beer ;)

Comment (olamy) [reply permissions/component-fips-bundle-test.yml:5]: I got few but still a [problem](https://github.com/jenkinsci/lib-fips-bundle-test/actions/runs/10848271899/job/30105055607) (not sure I can get more now :) ) 
I will wait tomorrow.

Comment (olamy) [reply permissions/component-fips-bundle-test.yml:5]: @timja 
still not working https://github.com/jenkinsci/lib-fips-bundle-test/actions/runs/10857213543/job/30133288891

Comment (olamy) [reply permissions/component-fips-bundle-test.yml:5]: I hope there is no mistake somewhere as I may have been confused by the naming convention :)
- github is `lib-fips-bundle-test`
- artifactId `fips-bundle-test`
-  file here is `component-fips-bundle-test.yml`

You guys made it hard to me to not make any mistake :)

Comment (olamy) [reply permissions/component-fips-bundle-test.yml:5]: issue created here https://github.com/jenkins-infra/helpdesk/issues/4290

**Pull request merged**